### PR TITLE
Update all.json from Allure Security findings

### PR DESCRIPTION
**Pull request by michalisFr**
These are the new sites for today and the corresponding top-level domains added to `all.json`.
Please keep in mind that the script checks against all.json from the master branch. If there are open PRs, there may be duplicates.
Also, please check if the domains added are any well-known that shouldn't be blacklisted at the top level (eg. netlify, ddns, etc) and change these to include the subdomain.

Please check them for eligibility and liveness, add the additional information (urlscan, screenshot) were applicable, and separate them in eligible, non-eligible and dead.
The curator should only copy the "eligible" ones into the spreadsheet.

Below are 50 new scam sites, along with 35 domains added to `all.json`:

| Site | Domain |
|------|--------|
| correct.zealtm.group | N/A |
| dapp.activationaccess.com | N/A |
| connect.ico-sale.com | N/A |
| support.bitvestnetwork.info | N/A |
| upgrade.synchronisedata.com | N/A |
| dapp.ibarachoir.com | N/A |
| dapp.easyseverconnect.com | N/A |
| sales.kubecoln.org | N/A |
| app.mainsecuretools.com | N/A |
| bot.dappwalletconnet.click | N/A |
| connect.sale-ico.com | N/A |
| wallcon.wdapwallreceivecondapp.live | N/A |
| connect.wallreceivecondapp.click | wallreceivecondapp.click |
| wallletuserconect.company | wallletuserconect.company |
| validationport.com | validationport.com |
| app.webswalletconnect.com | webswalletconnect.com |
| main.mainconnectedwallet.com | mainconnectedwallet.com |
| integrals-dexs.net | integrals-dexs.net |
| connect-mainnet01.ddns.us | ddns.us |
| bankk-11.cf | bankk-11.cf |
| integral-dex-bridges.net | integral-dex-bridges.net |
| fastautolinked.com | fastautolinked.com |
| sync.cryptoflips.net | N/A |
| lopezgarciaabogados.com | lopezgarciaabogados.com |
| walldmconecttkprecieve.live | walldmconecttkprecieve.live |
| connect-mainnet.ddns.us | N/A |
| vnwerolivh-3w4erhgv-38ih49-ghwehgfv-3w4g.obs.na-mexico-1.myhuaweicloud.com | myhuaweicloud.com |
| integral-dexs.com | integral-dexs.com |
| aliancrypto.com | aliancrypto.com |
| web.portalsync.live | portalsync.live |
| walletconnecti.com | walletconnecti.com |
| apps-syncs.net | apps-syncs.net |
| q2rv.com | q2rv.com |
| defisolutionprotocol.online | defisolutionprotocol.online |
| wallet.appsregistry.io | appsregistry.io |
| wcvalidate.dataconnectted.com | dataconnectted.com |
| m26securiter.info | m26securiter.info |
| wlt-register.live | wlt-register.live |
| trustwallet-verifyid-16.ml | trustwallet-verifyid-16.ml |
| connectiwallets.com | connectiwallets.com |
| wcsynchprotocols.com | wcsynchprotocols.com |
| mobilewalletsdapp.com | mobilewalletsdapp.com |
| connectwallet.waconlinkks.click | waconlinkks.click |
| app.connectdappsmove.com | N/A |
| ajudacef.com | ajudacef.com |
| naskizz.com | naskizz.com |
| migratev2finance.teachablei.com | teachablei.com |
| integrals-dex.com | integrals-dex.com |
| walletdappvalidator.io | walletdappvalidator.io |
| syncrvalidation.pagekite.me | pagekite.me |

### Eligible
| Site | Urlscan | Screenshot |
|-------|-------|-------|

### Non-eligible
| Site |
|-------|
|dapp.activationaccess.com|
|upgrade.synchronisedata.com|
|dapp.ibarachoir.com|
|dapp.easyseverconnect.com|
|bot.dappwalletconnet.click|
|connect.sale-ico.com|
|wallcon.wdapwallreceivecondapp.live|
|validationport.com|
|app.webswalletconnect.com|
|lopezgarciaabogados.com|
|walldmconecttkprecieve.live|
|web.portalsync.live|
|walletconnecti.com|
|ajudacef.com|
|naskizz.com|
|migratev2finance.teachablei.com|

### Dead
| Site |
|-------|
|correct.zealtm.group|
|connect.ico-sale.com|
|support.bitvestnetwork.info|
|sales.kubecoln.org|
|app.mainsecuretools.com|
|connect.wallreceivecondapp.click|
|wallletuserconect.company|
|main.mainconnectedwallet.com|
|integrals-dexs.net|
|integral-dex-bridges.net|
|fastautolinked.com|
|connect-mainnet01.ddns.us|
|bankk-11.cf|
|sync.cryptoflips.net|
|vnwerolivh-3w4erhgv-38ih49-ghwehgfv-3w4g.obs.na-mexico-1.myhuaweicloud.com|
|connect-mainnet.ddns.us|
|integral-dexs.com|
|aliancrypto.com|
|apps-syncs.net|
|q2rv.com|
|defisolutionprotocol.online|
|wallet.appsregistry.io|
|wcvalidate.dataconnectted.com|
|m26securiter.info|
|wlt-register.live|
|trustwallet-verifyid-16.ml|
|connectiwallets.com|
|wcsynchprotocols.com|
|mobilewalletsdapp.com|
|connectwallet.waconlinkks.click|
|app.connectdappsmove.com|
|integrals-dex.com|
|walletdappvalidator.io|
|syncrvalidation.pagekite.me|

### CODE REVIEW

**Comment by michalisFr** [comment]
@ical10 Third time's the charm! Here's my comment in the previous (failed) PR:
> This is the list of sites for today. Some of the first ones might also be in the previous two days, but I think I've fixed the script to not check those in the future.

**Comment by thwjanssen89** [comment]
Wow no errors!

File: all.json [hunk 1]
```diff
@@ -11055,6 +11055,41 @@
...
 		"zonefix.net",
-		"zwalletconnect.com"
+		"zwalletconnect.com",
+		"wallreceivecondapp.click",
+		"wallletuserconect.company",
+		"validationport.com",
+		"webswalletconnect.com",
+		"mainconnectedwallet.com",
+		"integrals-dexs.net",
+		"ddns.us",
```
Comment (jacogr) [inline all.json:11065]:
```suggestion
```

Should actually add this one to the auto-check list as "known top-level".

File: all.json [hunk 2]
```diff
@@ -11055,6 +11055,41 @@
...
+		"connectiwallets.com",
+		"wcsynchprotocols.com",
+		"mobilewalletsdapp.com",
+		"waconlinkks.click",
+		"ajudacef.com",
+		"naskizz.com",
+		"teachablei.com",
+		"integrals-dex.com",
+		"walletdappvalidator.io",
+		"pagekite.me"
```
Comment (jacogr) [inline all.json:11093]:
```suggestion
```

Also a known tunelling solution...

Comment (michalisFr) [reply all.json:11093]: And "myhuaweicloud.com" which redirects to "huaweicloud.com"

Comment (jacogr) [reply all.json:11093]: Sorry, missed that one, long day.

Comment (michalisFr) [reply all.json:11093]: For this we should add the full url:
`syncrvalidation.pagekite.me`

The bucket from "myhuaweicloud.com":
`vnwerolivh-3w4erhgv-38ih49-ghwehgfv-3w4g.obs.na-mexico-1.myhuaweicloud.com`
doesn't exist anymore, so I think we can safely leave that entry out entirely.

**Comment by ical10** [comment]
All right checking right now, gonna take a moment

**Comment by ical10** [comment]
well, they are either dead or non-eligible. Approving and merging.

File: all.json (hunk 1 above)
Comment (ical10) [reply all.json]: how do I add
`connect-mainnet01.ddns.us`
`connect-mainnet.ddns.us`
by myself?

**Comment by ical10** [comment]
All right, phew. Had to push commits with GH Desktop, but it worked.

**Review by ical10 — APPROVED** [review]

**Pull request merged**